src/Orchard.Cms.Web/Program.cs

```diff
@@ -1,20 +1,50 @@
 using System.IO;
+using System.Linq;
+using System.Reflection;
 using System.Threading;
+using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Modules;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
 using Orchard.Hosting;
+using Orchard.Logging;
 
 namespace Orchard.Cms.Web
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
-                .UseIISIntegration()
-                .UseKestrel()
-                .UseContentRoot(Directory.GetCurrentDirectory())
+            var host = WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration((hostingContext, config) =>
```

Comment:
Out of curiosity, why do you need to do all these things here? Adding the `appsettings` files or enabling the user secrets are things `CreateDefaultBuilder()` already does for you.